h/services/group_scope.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h.models import GroupScope
+from h.util import group_scope as scope_util
+
+
+class GroupScopeService(object):
+    def __init__(self, session):
+        self._session = session
+
+    def fetch_by_origin(self, uri):
```

Comment:
Is `fetch_by_origin()` intended to be a public function or is it only used by `fetch_by_scope()` below?